### DISCUSSION TITLE
Project roadmap

### DISCUSSION THREAD

**Discussion by metropt** [mebs/CM4-NAS#2] [General]
I did order some HDDs and a RPi4 to install OMV and nextcloud to use as a basic home NAS however one based on CM4 will be much better.

Can you share your roadmap for the project to understand if it is something I would like to work on?

Thank you

**Comment by mebs** [comment]
Of course! Converting this issue to a discussion, I'll update Readme.md with something a bit more useful.

**Reply by metropt** [reply]
I liked the way you designed your first board, taking out of the board what could cause problems so you can test it quickly.
Since it is supposed to be used with HDDs I think it is ok to have a bigger PCB, and that makes the layout of the board easier.

**Comment by mebs** [comment]
I updated the readme with a decent enough content to get started with, and I created a couple discussions in the idea section that touch on what can be done to improve the design.
The PCBs I have are a first iteration of a "final" product and I'll be happy with it If it works as it is right now, so I did not have a roadmap perse, just enhancements ideas. The rest will be fixing issues with the hardware as they come up and figuring out the software.
